web-client/integration-tests/utils/manuallyAddCaseToTrial.js

```diff
@@ -0,0 +1,26 @@
+// import { wait } from '../helpers';
+// import { withAppContextDecorator } from '../../src/withAppContext';
+
+export const manuallyAddCaseToTrial = cerebralTest => {
```

Comment:
It looks like there are a couple other functions that also manually add a case to a trial session:

`docketClerkManuallyAddsCaseToTrialSessionWithoutNote`
`petitionsClerkManuallyAddsCaseToTrial`
`petitionsClerkManuallyAddsCaseToTrialWithoutJudge`

Could any of those been reused?

Reply:
We tried using some of those helper files but they expect statement coded directly into the code, which makes them coded for a specific usecase which fails when we try to use them.  Instead of adding more arguments to the helper function to ignore the expect statements, we went with a new function that just adds the case to the trial and doesn't expect anything.  This allows us to keep the expect statement in the test file itself instead of helpers failing because we are using them for the wrong purpose.